### DISCUSSION TITLE
Error Compiling from Source Code

### DISCUSSION THREAD

**Discussion by NiLMoXiE** [mriscoc/Ender3V2S1#435] [Q&A] [answered]
Trying to compile a 4.2.2 version with CR Touch and UBL along with PT1000 thermistor (E3D), I used the configurations for Ender3V2-422-BLT

I changed #define TEMP_SENSOR_0 to 1047

I disabled AUTO_BED_LEVELING_BILINEAR and enabled AUTO_BED_LEVELING_UBL as soon as I change this I get the #include errors

Getting the following 4 errors on compiling:

![Screenshot 2022-10-09 141902](https://user-images.githubusercontent.com/110064677/194759165-e23239e5-c8a6-4a99-9b98-df3542c1562b.png)

![Screenshot 2022-10-09 142424](https://user-images.githubusercontent.com/110064677/194759384-65ab6215-b4e7-4c59-8230-4b4fd0ea6071.png)

Any help would be much appreciated.

**Comment by mriscoc** [comment, marked as answer]
Use the configuration files from https://github.com/mriscoc/Special_Configurations as templates.

**Reply by NiLMoXiE** [reply]
Thankyou
